devsetup/Makefile

```diff
@@ -84,6 +84,18 @@ crc_attach_default_interface_cleanup: ## Detach default libvirt network from CRC
 	virsh --connect=qemu:///system detach-interface crc network --mac "$$MAC_ADDRESS"
 	-virsh --connect=qemu:///system net-update default delete ip-dhcp-host "<host name='crc'/>" --config --live
 	sleep 5
+edpm_compute_baremetal: export BMAAS_INSTANCE_MEMORY=16384
+edpm_compute_baremetal: export BMAAS_INSTANCE_VCPUS=8
+edpm_compute_baremetal: export BMAAS_INSTANCE_DISK_SIZE=40
+edpm_compute_baremetal: export PROVISIONING_INTERFACE=crc-bmaas
+edpm_compute_baremetal: export BMAAS_NETWORK_IPADDRESS=172.22.0.3
+edpm_compute_baremetal: export BMAAS_BRIDGE_IPADDRESS=172.22.0.2
+edpm_compute_baremetal:
+	$(eval $(call vars))
+	pushd .. && make nmstate openstack NETWORK_ISOLATION=false
+	make bmaas
+	scripts/edpm_compute_bmaas.sh
```

Comment:
~~~
scripts/edpm_compute_bmaas.sh
make: scripts/edpm_compute_bmaas.sh: No such file or directory
make: *** [Makefile:99: edpm_compute_baremetal] Error 127
~~~
it shoud be scripts/edpm-compute-bmaas.sh

Reply:
thanks